jest.config.js

```diff
@@ -17,7 +17,7 @@ const processStdoutWrite = process.stdout.write.bind(process.stdout);
 process.stdout.write = (str, encoding, cb) => {
     // Core library will directly call process.stdout.write for commands
     // We don't want :: commands to be executed by the runner during tests
-    if (!str.match(/^::/)) {
+    if (!String(str).match(/^::/)) {
```

Comment:
jest new version was throwing error as type for str here was being recognized as "any". Implicitly converting to String fixed this.